modules/drivers/oracle/test/metabase/test/data/oracle.clj

```diff
@@ -38,22 +38,28 @@
 (defn- connection-details []
   (let [details* {:host         (tx/db-test-env-var-or-throw :oracle :host "localhost")
                   :port         (Integer/parseInt (tx/db-test-env-var-or-throw :oracle :port "1521"))
-                  :user         (tx/db-test-env-var-or-throw :oracle :user "system")
-                  :password     (tx/db-test-env-var-or-throw :oracle :password "password")
+                  :user         (tx/db-test-env-var :oracle :user)
+                  :password     (tx/db-test-env-var :oracle :password)
```

Comment:
For the Oracle 21.3 tests SSL authentication is used, no user name or password is provided.